common/remote/sockfile.cpp

```diff
@@ -4003,6 +4011,10 @@ class CRemoteFileServer : public CInterface, implements IRemoteFileServer
         // can revert to previous behavior with conf file setting "allow_pgcache_flush=false"
         if (extraFlags == IFEnone)
             extraFlags = IFEnocache;
+        // also try to recv extra flags
+        unsigned eflags = IFUnone;
+        if (msg.remaining() >= sizeof(unsigned))
```

Comment:
This serialize info. can only exist in clients that have also sent 'extra' flags (as added long after that code was added), but from this server code a casual glance would suggest it could fail the previous 'if (msg.remaining() >= sizeof(byte))' test, but potentially succeed here.
It would be clearer if there was a comment to explain and/or it this deserialization was moved under the 'extra' branch.